.github/workflows/ci-dgraph-jepsen-tests.yml

```diff
@@ -0,0 +1,67 @@
+name: ci-dgraph-jepsen-tests
```

Comment:
I think we need to simplify this workflow:
1. We should probably run it once or twice in a week
2. Do we know that it passes today?
3. We don't need 3 different steps (cluster up, running the test, cluster down), just one command can do it all
4. We also may not want to run it on every PR, we should just run it periodically

Reply:
1. Agreed
2. Yes, you can see the job completed successfully
3. Not sure the reasoning here?  This is how the jepsen command is set up, and is similar to how we run other jobs
4. Agreed, will modify to only run on schedule

Reply:
oh, wow! We still pass jepsen tests 🏖️ 
For 3rd, my reasoning was that when `main.go` handles setting up the cluster and destroying it, what is the benefit we get in separating it into 3 steps? when I run ` ./jepsen --jepsen-root ${{ env.JEPSEN_ROOT }} --web=false --test-all -l 300`, doesn't it set up and destroy the cluster?

Also, would it be a bad idea to keep the time limit to 600, the default?